docs/src/main/asciidoc/grpc-generation-reference.adoc

```diff
@@ -159,7 +165,7 @@ However, it's recommended to use Quarkus support unless you have a specific need
 
 To do this, define the following properties in the `<properties>` section:
 
-[source,xml,subs="attributes"]
+[source,xml,subs=attributes+]
```

Comment:
In this very case, it was dropping the XML tags and just output the unevaluated Asciidoc attributes.